build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -x
+set -o errexit
+set -o pipefail
+
+# Get a clean checkout
+git fetch origin || true
+git checkout origin/master || true
+git reset HEAD || true
+git checkout -- || true
+
+# Resolve pending metadata
+./main
```

Comment:
I see `./main` is checked into the repo. It should be built in the CI workflow instead, right?